bemserver_core/database.py

```diff
@@ -134,13 +136,25 @@ def delete(self):
         """Delete object"""
         db.session.delete(self)
 
+    def _before_flush(self):
```

Comment:
A docstring would be more informative and would remove the need for the pass statement.